src/component/setup/index.js

```diff
@@ -53,8 +53,9 @@ export default function (component, config) {
   // // setup watchers
   if (config.watch) setupWatch(component, config.watch)
 
-  // // setup routes
-  if (config.routes) setupRoutes(component, config.routes)
+  // // setup router
+  const routerConfig = config.router !== undefined ? config.router : config.routes !== undefined ? config.routes : undefined
```

Comment:
is this part `config.routes !== undefined ? config.routes : undefined` really needed here? 🤔